pkgs/development/python-modules/dipy/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, packaging
+, pytest
+, python
+, cython
+, numpy
+, scipy
+, pandas
+, scikitlearn
+, statsmodels
+, h5py
+, tables
+, cvxpy
+, matplotlib
+, nibabel
+, withCvxpy ? false  # unfree dependency on `mkl`
+}:
+
+buildPythonPackage rec {
+  pname = "dipy";
+  version = "1.1.1";
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner  = "nipy";
+    repo   = pname;
+    rev    = version;  # "1fb90ab8bbec66bdf26c59cc04a90df3cfc88510";
+    #sha256 = "08sfxb45fvf8g165skwq1ziig98h562jiqp49855yhgv2m5l3icg";
+    sha256 = "08abx0f4li6ya62ilc59miw4mk6wndizahyylxhgcrpacb6ydw28";
+  };
+
+  buildInputs = [ cython ];
+  propagatedBuildInputs = [
+    packaging
+    numpy
+    scipy
+    pandas
+    h5py
+    nibabel
+    tables
+    matplotlib
+    scikitlearn
+    statsmodels
+  ] ++ lib.optional withCvxpy cvxpy;
+
+  checkInputs = [ pytest ];
+
+  # disable tests for now due to:
+  #   - some tests require data download (see dipy/dipy/issues/2092);
+  #   - running the tests manually causes a multiprocessing hang;
+  #   - import weirdness when running the tests
+  doCheck = false;
```

Comment:
would still prefer tests... but it looks like their test suites require a lot of bootstrapping, so I'm fine with this

Reply:
It looks like part of the problem might be that `nose` is propagated by `matplotlib` which I had as a dependency, and this was causing `pytest --ignore=...` to do nothing.  So it might still be possible to enable with some more work ...